webpack.config.client.js

```diff
@@ -28,7 +28,7 @@ module.exports = ({ resolvePath, IS_CI, IS_PROD, START_DEV_SERVER }) => {
       filename: 'static/js/[name].[hash:8].js',
       // need full URL for dev server & HMR: https://github.com/webpack/docs/wiki/webpack-dev-server#combining-with-an-existing-server
       publicPath: IS_PROD
-        ? `${process.env.SIMORGH_BASE_URL}/`
+        ? `${process.env.PUBLIC_PATH}/`
```

Comment:
As spoken about in person would be good to rename this to be more explicit such as `SIMORGH_PUBLIC_STATIC_ASSETS_PATH` and then make it dependant on https://github.com/bbc/simorgh-infrastructure/pull/192/files